pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-facebook/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, pidgin, glib, json_glib } :
+
+stdenv.mkDerivation rec {
+  name = "purple-facebook-${version}";
+  version = "66ee77378d82";
```

Comment:
In the case of a commit version should be the date of that commit i.e. probably release in this case(?):

> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23". - [nixpkgs manual - 10.2. Package naming](https://nixos.org/nixpkgs/manual/#sec-package-naming)

The problem is that these release versions could start with a character while the manual states:

> The version part of the name attribute must start with a digit (following a dash) — e.g., "hello-0.3.1rc2".